recipes/pedcheck/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pedcheck" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haraldgrove/{{ name }}/archive/master.zip
+  sha256: 239a9c31793cab0f4e4374ba01d174c018f836b41867870d493663c16f78fcc3
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
The problem is, this repository does not have a ``setup.py``, so ``pip`` will not install anything and it will raise an error